packages/help-extension/src/index.tsx

```diff
@@ -34,7 +34,7 @@ const RESOURCES = [
  * The command IDs used by the help plugin.
  */
 namespace CommandIDs {
-  export const open = 'help:open';
+  export const open = 'help:open-tab';
```

Comment:
This is temporarily required since the upstream plugin already defines the `help:open` command, and would fail to activate if it already exists.

Reply:
Opened https://github.com/jupyterlab/jupyterlab/pull/14837